cmc-domain-sample-data/build.gradle

```diff
@@ -0,0 +1,13 @@
+apply plugin: 'java'
+
+sourceCompatibility = 1.8
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+  compile project(':cmc-domain-model')
+
+  testCompile group: 'junit', name: 'junit', version: '4.12'
```

Comment:
Is this needed for sample data project? Are we testing sample data somehow?

Reply:
Not required. Removed.